cmd/cloud-controller-manager/.import-restrictions

```diff
@@ -41,6 +41,6 @@ rules:
       - k8s.io/kubernetes/pkg/util/taints
       - k8s.io/kubernetes/pkg/proxy/util
       - k8s.io/kubernetes/pkg/proxy/util/testing
-      - k8s.io/kubernetes/pkg/util/slice
+      - k8s.io/kubernetes/pkg/util/finalizers
```

Comment:
I'm not sure one generic function to remove a string from a slice of strings should be put into a new package with this name. I'd suggest to leave it in the slice package.

Reply:
Moved the changes back :)